.travis.yml

```diff
@@ -1,43 +1,43 @@
+dist: xenial
 language: rust
 matrix:
   include:
     - os: linux
       rust: nightly
-      env: GTK=3.4
+      env: GTK=3.14 FEATURES=
```

Comment:
Isn't something missing after `FEATURES=`?

Reply:
it's empty because we build with the default features :)

Reply:
Ok!

Reply:
`cargo test --features "$FEATURES""` looks strange but seems cargo allow it